docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contributing to Devworkspace-Operator
+
+Hello there! Thank you for choosing to the contributing to devfile/devworkspace-operator. Navigate through the following to understand more about contributing here.
+
+- [Contributing to Devworkspace-Operator](#contributing-to-devworkspace-operator)
+- [How to Contribute](#how-to-contribute)
+  - [Set up your Development Environment](#set-up-your-development-environment)
+      - [Running the controller locally.](#running-the-controller-locally)
+  - [Testing Changes](#testing-changes)
+      - [Test run the controller](#test-run-the-controller)
+      - [Developing Webhooks](#developing-webhooks)
+  - [Signing-off on Commits](#signing-off-on-commits)
+
+# How to Contribute
+To contribute to the devworkspace-operator project, developers should follow the fork and pull request workflow. 
+## Set up your Development Environment
+
+
+**1.** Fork [Devworkspace operator](https://github.com/devfile/devworkspace-operator) repository.
```

Comment:
Here, since we are referring to the repository itself, we can probably write "devworkspace-operator"